tests/suite/client_settings_test.go

```diff
@@ -91,9 +91,130 @@ var _ = Describe("ClientSettingsPolicy", Ordered, Label("functional", "cspolicy"
 			}
 		})
 
+		Context("nginx config", func() {
+			var conf *framework.Payload
+
+			BeforeAll(func() {
+				podNames, err := framework.GetReadyNGFPodNames(k8sClient, ngfNamespace, releaseName, timeoutConfig.GetTimeout)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(podNames).To(HaveLen(1))
+
+				ngfPodName := podNames[0]
+
+				conf, err = resourceManager.GetNginxConfig(ngfPodName, ngfNamespace)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			DescribeTable("is set properly for",
+				func(expCfgs []framework.ExpectedNginxField) {
+					for _, expCfg := range expCfgs {
+						failureMsg := fmt.Sprintf(
+							"directive '%s' with value '%s' not found in expected place",
+							expCfg.Key, expCfg.Value,
+						)
+						Expect(framework.ValidateNginxFieldExists(conf, expCfg)).To(BeTrue(), failureMsg)
+					}
+				},
+				Entry("gateway policy", []framework.ExpectedNginxField{
+					{
+						Key:                   "include",
+						Value:                 "gw-csp.conf",
```

Comment:
Why omit the namespace from the file? In case we change the namespace in the future?

Reply:
We could include it, I don't see an issue with that. I think I wasn't sure what that value in the filename was, so if we do include it, let's be clear that it's the namespace.

Reply:
I think we should be more specific when possible. It would also be nice to have the whole filename when debugging.